### ISSUE TITLE
  About the issue of non-decreasing training loss

### ISSUE THREAD

**Issue by continue1213** [IceClear/StableSR#120]
When training the Time-aware encoder with SFT, the training loss does not decrease at all. How should I address this situation?
The training statement I executed is: python main.py --train --base configs/stableSRNew/v2-finetune_text_T_512.yaml --gpus 2,3 --name stable_sr_face02 --scale_lr False

**Comment by Luciennnnnnn** [comment]
> The loss logged out is the loss at the random step. Generally, the loss at a large step, e.g., step 800 is certainly bigger than the loss at later steps, e.g., step 50. So the loss fluctuates from your view.

Intuitively, the loss should decrease in terms of long period (though fluctuates in short period). I found it is hard to train diffusion models for SR, could you please provide a loss curve plot as a reference? Thanks!
